packages/url/src/filter-url-for-display.js

```diff
@@ -21,7 +21,7 @@ export function filterURLForDisplay( url, maxLength = null ) {
 		filteredURL = filteredURL.replace( '/', '' );
 	}
 
-	const mediaRegexp = /([\w|:])*\.(?:jpg|jpeg|gif|png|svg)/;
+	const mediaRegexp = /\/([^\/?]+)\.(?:[\w]+)(?=\?|$)/;
```

Comment:
Small point, but I wonder if it's worth renaming this regex now that it's not explicitly looking for "media" any more?

Reply:
Good point!